modules/machine-config-overview.adoc

```diff
@@ -27,6 +27,12 @@ you can set up a custom pool of worker nodes that includes particular hardware
 features needed by an application. However, examples in this section focus
 on changes to the default pool types.
 
+[IMPORTANT]
+====
+A node can have multiple labels applied that indicate it is "master", "worker", etc,
```

Comment:
> 🚫  etc. n
Do not use. Use “and so on” when you list a clear sequence of elements, such as “1, 2, 3, and so on” or “Monday, Tuesday, Wednesday, and so on.”

Since this isn't a clear sequence, maybe:
```suggestion
A node can have multiple labels applied that indicate its type, such as `master` or `worker`,
```